ecmd-core/dll/ecmdDllCapi.C

```diff
@@ -6369,16 +6369,13 @@ std::string dllParseReturnCode(uint32_t i_returnCode) {
   std::string l_version = "default";
   uint32_t rc = dllQueryFileLocationHidden2(dummy, ECMD_FILE_HELPTEXT, paths, l_version);
 
-  // Assume for now we only have one helptext path returned
-  filePath = paths.begin()->textFile;
-
-  if (rc || (filePath.length()==0)) {
+  if (rc || paths.empty()) {
```

Comment:
Do we want to keep the original check in there too if it gets past empty()?
```
if (rc || paths.empty() || (paths.begin()->textFile.length() == 0)) {
```

Reply:
I couldn't tell what the `.length()` check would be good for hre.  I think it's an artifact of the old version of this function where it didn't return a list, just a string - and a check of an empty string would have been a decent thing.  I can't see a reason someone would push something empty onto the list.

Reply:
I'll change it though just to prevent a possible coredump